src/test/java/de/tum/in/www1/artemis/AbstractSpringIntegrationJenkinsGitlabTest.java

```diff
@@ -89,7 +90,13 @@ public void mockConnectorRequestsForSetup(ProgrammingExercise exercise) throws E
     }
 
     @Override
-    public List<Verifiable> mockConnectorRequestsForStartParticipation(ProgrammingExercise exercise, String username, Set<User> users) throws IOException, URISyntaxException {
+    public void mockCopyRepositoryForParticipation(ProgrammingExercise exercise, String username, HttpStatus status) throws URISyntaxException, IOException {
```

Comment:
The codacy point here I believe is valid. If you make this method abstract you can ensure that it is not called accidentally while it is not implemented.

Reply:
This is intentional. This is a requirement by the interface that is implemented. This method has to be implemented later on, when we implement the tests for Jenkins